pkg/common-controller/snapshot_controller.go

```diff
@@ -671,6 +672,16 @@ func (ctrl *csiSnapshotCommonController) createSnapshotContent(snapshot *crdv1.V
 		},
 	}
 
+	nodeName, err := ctrl.getManagedByNode(volume)
```

Comment:
Why do we need to get nodeName if distributed snapshotting is not supported? I think this is going have impact on deployment that does have support distributed snapshotting as well.

Reply:
We could probably use a flag for snapshot controller to indicate whether we are using distributed snapshotting or not. But, since only a single implementation of snapshot controller will be deployed even in case of distributed snapshotting, I thought a flag like this might cause confusion.

 Let me know what you think. I could add a flag if it impacts the deployment that doesn't support distributed snapshotting.

Reply:
Yes, please add a flag to indicate distributed snapshotting is supported and set it to false by default.

Reply:
Added the flag to indicate that distributed snapshotting is enabled.